src/book/03-guides/04-general/01-git.mdx

```diff
@@ -42,6 +42,8 @@ To be approved, you should add reviewers to the pull request. If you have a proj
 
 ### Code Review
 
+When making a PR you can select a number of reviewers to review your code, they will get a notification. Please choose people that you feel will be able to understand the problem your code is trying to solve.
```

Comment:
This overlaps a lot with the previous paragraph. Probably should consolidate them into one paragraph, I think.
Two suggestions for this paragraph anyway :p

```suggestion
When making a PR you can select a number of reviewers to review your code, who each get a notification. Choose people who will be able to understand the problem your code is trying to solve.
```
Or
```suggestion
When making a PR you can select a number of reviewers to review your code, who each get a notification. Ideally, choose people that you feel will be able to understand the problem your code is trying to solve.
```